plan/service.py

```diff
@@ -126,18 +126,12 @@ def available_plans(self, owner: Owner) -> List[PlanData]:
         if owner and sentry.is_sentry_user(owner=owner):
             available_plans += SENTRY_PAID_USER_PLAN_REPRESENTATIONS.values()
 
```

Comment:
Should we be using `plan_activated_users` or `plan_user_count`? Say they have purchased 100 seats but only 10 have been activated, do we want to show Team plan as an option?

We had to switch to `plan_activated_users` from `plan_user_count` because trial plan sets the `plan_user_count` to some fixed large number, so we just used `plan_activated_users`. Now that we want to surface this plan to non-trial plans, how do we want to handle this?

Reply:
We still wanna go with activated users, here's what AJ said:

> I believe we want to stick with plan activated users - my thought process on this is that this is who is actually using the product, if you only have 10 users I’d want them to be on the most economically sensible plan for them, even if it means a downgrade from pro to team - same logic applies when canceling, even if they reduce # of users AND downgrade to Team that’s still preferable to the alternative of giving them the less activated save offer (30% off pro) and having them maybe choose Team